app/views/users/_card.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="card shadow rounded-lg">
+  <div class="card-body">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs">
+          <img src="//lh3.googleusercontent.com/-6V8xOA6M7BA/AAAAAAAAAAI/AAAAAAAAAAA/rzlHcD0KYwo/photo.jpg?sz=120" class="rounded-circle" style="width: 50px">
+        </div>
+        <a class="col-8" href=<%= user_path(user) %>>
+          <h5 class="card-title"><%= user.name %></h5>
```

Comment:
Maybe it would be nice to use the `display_name` here

Reply:
it is the display_name now